libs/ui/src/export_logs_modal.test.tsx

```diff
@@ -65,7 +67,7 @@ test('renders loading screen when usb drive is mounting or ejecting in export mo
     );
     userEvent.click(screen.getByText('Save Log File'));
     await screen.findByText('Loading');
-    userEvent.click(screen.getByText('Cancel'));
+    userEvent.click(await screen.findByText('Cancel'));
```

Comment:
Since React 18 is about concurrency and automatic batching of state updates, it means that some things are faster or slower in tests. The largest category of test tweaks is around properly waiting for things on the page _or_ finding workarounds to test things that are now too fast to catch on screen by default.

Reply:
I think this particular example was only working by chance before.